pkg/migrations/op_add_column_test.go

```diff
@@ -255,6 +273,58 @@ func TestAddColumn(t *testing.T) {
 				}, res)
 			},
 		},
+		{
+			name: "add column to non-empty table without primary key",
+			migrations: []migrations.Migration{
+				{
+					Name: "01_add_table",
+					Operations: migrations.Operations{
+						&migrations.OpCreateTable{
+							Name: "users",
+							Columns: []migrations.Column{
+								{
+									Name: "name",
+									Type: "varchar(255)",
+								},
+							},
+						},
+					},
+				},
+				{
+					Name: "02_add_column",
+					Operations: migrations.Operations{
+						&migrations.OpAddColumn{
+							Table: "users",
+							Column: migrations.Column{
+								Name:     "age",
+								Type:     "integer",
+								Nullable: true,
+							},
+							Up: "1",
+						},
+					},
+				},
+			},
+			afterRollback: func(t *testing.T, db *sql.DB, schema string) {
+				// The new column has been dropped from the underlying table
+				columnName := migrations.TemporaryName("age")
+				ColumnMustNotExist(t, db, schema, "users", columnName)
+
+				// The table's column count reflects the drop of the new column
+				TableMustHaveColumnCount(t, db, schema, "users", 1)
+
+				// Insert a row to make sure we get a not possible error because there's no identity column
```

Comment:
```suggestion
				// Insert a row to make sure we get a `backfill.NotPossible` error when the migration is restarted, because there's no identity column
```